VoxelSniperCore/src/main/java/com/github/kevindagame/brush/CloneStampBrush.java

```diff
@@ -36,8 +32,8 @@ public class CloneStampBrush extends StampBrush {
     private void clone(final SnipeData v) {
         VoxelLocation point = getTargetBlock().getLocation().makeMutable();
         point.add(0, v.getcCen(), 0);
-        this.startingPoint = point.makeImmutable();
-        var positions = Shapes.cylinder(startingPoint, RotationAxis.Y, v.getBrushSize(), v.getVoxelHeight(), 0, false);
+        BaseLocation startingPoint = point.makeImmutable();
```

Comment:
Is there a reason that you make the point immutable? I don't think it's necessary here

Reply:
Good point